pom.xml

```diff
@@ -51,6 +51,15 @@ under the License.
 		</developerConnection>
 	</scm>
 
+	<pluginRepositories>
+		<pluginRepository>
+			<!-- Allows exec-maven-plugin to resolve snapshot plugin dependencies -->
```

Comment:
Basically, plugins set on the dependency are only looked up in specific repositories, and apparently not in those that we have configured :shrug: